glean-core/metrics.yaml

```diff
@@ -391,3 +391,17 @@ glean.upload:
     expires: never
     no_lint:
       - COMMON_PREFIX
+
+glean.database:
+  size:
+    type: memory_distribution
+    description: >
+      The size of the database file at startup.
+    memory_unit: kilobyte
```

Comment:
Shouldn't this be `byte` since [len returns bytes](https://doc.rust-lang.org/std/fs/struct.Metadata.html#method.len) and the [accumulate](https://searchfox.org/glean/rev/85c44c5f0bb947803afdc28563bc3b5c7296d7a5/glean-core/src/metrics/memory_distribution.rs#63-66) expects data in the same unit we declare here?

Reply:
I ... had that before and I have no idea why I changed it 🤔